#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_about:plugins.dpatch by Mike Hommey <glandium@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Install the files for about:plugins. bz#328678

@DPATCH@

=== toolkit/content/jar.mn
==================================================================
--- xulrunner/toolkit/content/jar.mn	(revision 209)
+++ xulrunner/toolkit/content/jar.mn	(local)
@@ -78,4 +78,4 @@
 +  content/global/selectDialog.xul            (selectDialog.xul)
 *+ content/global/viewZoomOverlay.js          (viewZoomOverlay.js)
 #endif
-
+   content/global/plugins.html                (../../xpfe/global/resources/content/plugins.html)
=== toolkit/themes/winstripe/global/jar.mn
==================================================================
--- xulrunner/toolkit/themes/winstripe/global/jar.mn	(revision 209)
+++ xulrunner/toolkit/themes/winstripe/global/jar.mn	(local)
@@ -153,3 +153,4 @@
         skin/classic/global/console/console-error-caret.gif         (console/console-error-caret.gif)
         skin/classic/global/console/console-error-dash.gif          (console/console-error-dash.gif)
 #endif
+        skin/classic/global/plugins.css                             (../../../../themes/classic/global/plugins.css)
